client/containers/DashboardReviews/DashboardReviews.tsx

```diff
@@ -66,6 +76,11 @@ const DashboardReviews = (props: Props) => {
 													<div className="number">
 														R{review.number}: {review.title}
 													</div>
+													{hasReviewers && (
+														<div className="note">
+															by {parseReviewers(reviewers)}&nbsp;
```

Comment:
IMO this function should be called `renderReviewerByline` or something similar since no actual parsing, ie transforming data into a higher level representation, occurs within